src/components/App/index.tsx

```diff
@@ -246,10 +248,15 @@ export default class App extends Component<Props, State> {
       loadedCounter: loadingCounter,
     };
 
-    this.setState({ images, error: '' });
+    this.setState({ images });
+  }
+
+  showError (error: string) {
+    if (!this.snackbar) return;
```

Comment:
Nit: Throw instead? This shouldn't happen, right?

Reply:
I guess it could happen if an error occurred prior to rendering, but it doesn't matter to me